cypress/integration/inventory/Item-status-date-updates.spec.js

```diff
@@ -0,0 +1,376 @@
+          user.userId, effectiveLocationServicePoint.body.id);
+        Locations.createLocationViaApi(effectiveLocation)
+          .then(locations => {
+            orderLine.locations[0].locationId = locations.body.id;
+            itemLocation = locations.body.name;
+          });
+        cy.login(userProperties.username, userProperties.password)
+          .then(() => {
+            Orders.createOrderWithOrderLineViaApi(order, orderLine)
+              .then(orderNumber => {
+                createdOrderNumber = orderNumber;
+              });
+          });
+        cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+          .then((users) => {
+            userBarcode = users[0].barcode;
+          });
+      });
+
+    cy.createTempUser([
+      permissions.checkoutAll.gui,
+      permissions.requestsAll.gui,
+    ])
+      .then(userProperties => {
+        userForDeliveryRequest = userProperties;
+        cy.getUsers({ limit: 1, query: `"username"="${userForDeliveryRequest.username}"` })
+          .then((users) => {
+            userForDeliveryRequest.barcode = users[0].barcode;
+            userRequestPreferences.userId = users[0].id;
+            cy.getAddressTypesApi({ limit: 1 }).then(addressTypes => {
+              userRequestPreferences.defaultDeliveryAddressTypeId = addressTypes[0].id;
+            })
+              .then(() => {
+                UpdateUser.updateUserAddress(users[0], [{ city: 'New York',
+                  addressTypeId: userRequestPreferences.defaultDeliveryAddressTypeId,
+                  countryId: 'US' }]);
+                cy.createUserRequestPreferencesApi(userRequestPreferences);
+              });
+          });
+      });
+  });
+
+  afterEach(() => {
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${userItemBarcode}"` })
+      .then((instance) => {
+        instance.items.forEach((item) => {
+          cy.deleteItem(item.id);
+        });
+        cy.deleteHoldingRecord(instance.holdings[0].id);
+        cy.deleteInstanceApi(instance.id);
+      });
+    cy.deleteOrderApi(order.id);
+    cy.deleteUser(user.userId);
+    cy.deleteServicePoint(notEffectiveLocationServicePoint.body.id);
+    // TODO delete servicePoints
+    Requests.getRequestApi({ limit: 1, query: `"item.barcode"=="${userItemBarcode}"` })
+      .then(requests => {
+        requests.forEach(request => {
+          Requests.deleteRequestApi(request.id);
+        });
+      });
+  });
+
+  it('C9200 Item status date updates', { tags: [TestTypes.smoke] }, () => {
+    const caption = 'autotestCaption';
+    const numberOfPieces = '3';
+
+    cy.visit(TopMenu.ordersPath);
+    Orders.searchByParameter('PO number', createdOrderNumber);
+    Helper.selectFromResultsList();
+    Orders.openOrder();
+    OrdersHelper.verifyOrderDateOpened();
+    cy.visit(TopMenu.inventoryPath);
+    InventorySearch.searchByParameter('Title (all)', instanceTitle);
```

Comment:
this portion of code is repeated. Try to wrap in related fragment or into this test